queries/c/injections.scm

```diff
@@ -9,7 +9,7 @@
   (#set! injection.language "re2c"))
 
 ((comment) @injection.content
-  (#lua-match? @injection.content "/[*][!<*][^a-zA-Z]")
+  (#lua-match? @injection.content "/[*\/][!*\/][^a-zA-Z]")
```

Comment:
```suggestion
  (#lua-match? @injection.content "/[*\/][!<*\/][^a-zA-Z]")
```

Reply:
`//<` is not a doxygen comment.

Reply:
https://www.doxygen.nl/manual/docblocks.html#memberdoc

Reply:
```suggestion
  (#lua-match? @injection.content "/[*\/][!*\/]<?[^a-zA-Z]")
```